jina/serve/runtimes/asyncio.py

```diff
@@ -193,3 +206,7 @@ def _log_data_request(self, request: DataRequest):
         self.logger.debug(
             f'recv DataRequest at {request.header.exec_endpoint} with id: {request.header.request_id}'
         )
+
+    @property
+    def _flow_id(self):
+        return None
```

Comment:
I think it's better to either rename to `pod_id` or have another function like _get_telemetry_extra_kwargs which is applicable to all runtimes in general

Reply:
I want it to be a `Flow` representing ID. Not generally `pod_id`. `Flow_id` can be interpreted by us, while `pod_id` not